README.md

```diff
@@ -129,10 +129,13 @@ This information is for project maintainers:
 
 Instructions
 
-- Use [`npm version`](https://docs.npmjs.com/cli/version) to update the version.
-- Commit and push the changes.
-- Release to npmjs.com by triggering a Shipit deploy
-- Create a release on GitHub (Shipit will tag the release automatically).
+- Use [`npm version <major|minor|patch>`](https://docs.npmjs.com/cli/version) to update the version in `package.json` and `package-lock.json`.
+- Push the changes and the tag with `git push --follow-tags`
+- Open new PR corresponding to the new release.
+- Once approved, merge the PR.
+- Release to npmjs.com by triggering a Shipit deploy of the master branch.
```

Comment:
I think merging the PR can be mentioned either in the step before or after, no need to have a separate step just for it

Reply:
👍